luigi_pipeline/tests/test_seqr_loading_tasks.py

```diff
@@ -57,10 +57,11 @@ def test_seqr_loading_validate_match_noncoding_only(self, mock_sample_type_stats
     def test_seqr_loading_validate_wes_mismatch(self, mock_sample_type_stats, mock_contig_check):
         # Supposed to be WES but we report as WGS.
         mock_sample_type_stats.return_value = self._sample_type_stats_return_value(0, 0, False, 0, 0, True)
-        self.assertRaises(SeqrValidationError, SeqrVCFToMTTask.validate_mt, self.test_mt, '37', 'WGS')
+        self.assertRaisesRegex(SeqrValidationError, "specified as WGS but appears to be WES", SeqrVCFToMTTask.validate_mt, self.test_mt, '37', 'WGS')
```

Comment:
Oh does the above formatting have to do with this test?

Reply:
Nope!  It should work fine as is.